SimeonInstaller.ps1

```diff
@@ -969,6 +969,41 @@ CRLFOption=CRLFAlways
         }
 
         Install-SimeonTenantPipeline -Organization $Organization -Project $Project -Name $Name -Credential $Credential -PipelineVariables $PipelineVariables @environmentArgs
+
+        Install-SimeonTenantLibrary -Organization $Organization -Project $Project
+    }
+
+    <#
+    .SYNOPSIS
+    Creates/updates a shared library for a tenant in Azure DevOps
+    #>
+    function Install-SimeonTenantLibrary {
+        [Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSReviewUnusedParameter', '', Scope = 'Function')]
+        [CmdletBinding()]
+        param(
+            # The Azure DevOps organization name
+            [ValidateNotNullOrEmpty()]
+            [string]$Organization,
+            # The project name in DevOps
+            [ValidateNotNullOrEmpty()]
+            [string]$Project = 'Tenants'
+        )
+
+        $token = Get-SimeonAzureDevOpsAccessToken -Organization $Organization -Project $Project
```

Comment:
if ($Project.Contains(" ")) {
            throw “Project name must not contain spaces”
        }

Reply:
Run VS Code formatting (alt+shift+f)